tests/extract-dfns.js

```diff
@@ -137,38 +137,6 @@ const tests = [
    html: "<dfn data-lt='foo \n   |\nbar' id=foo>Foo</dfn>",
    changesToBaseDfn: [{linkingText: ["foo", "bar"]}]
   },
-  {title: "marks as public a <dfn data-export>",
-   html: "<dfn id=foo data-export>Foo</dfn>",
-   changesToBaseDfn: [{access: 'public'}]
-  },
-  {title: "marks as public a <dfn data-dfn-type='interface'>",
-   html: "<dfn id=foo data-dfn-type=interface>Foo</dfn>",
-   changesToBaseDfn: [{access: 'public', type: 'interface'}]
-  },
-  {title: "marks as private a <dfn data-noexport data-dfn-type='interface'>",
-   html: "<dfn id=foo data-noexport data-dfn-type=interface>Foo</dfn>",
-   changesToBaseDfn: [{type: 'interface'}]
-  },
-  {title: "detects informative definitions",
-   html: "<div class=informative><dfn id=foo>Foo</dfn></div>",
-   changesToBaseDfn: [{informative: true}]
-  },
-  {title: "associates a definition to a namespace",
-   html: "<dfn data-dfn-for='Bar,Baz' id=foo>Foo</dfn>",
-   changesToBaseDfn: [{for:['Bar', 'Baz']}]
-  },
-  {title: "considers definitions in headings",
-   html: "<h2 data-dfn-type=dfn id=foo>Foo</h2>",
-   changesToBaseDfn: [{}]
-  },
-  {title: "ignores elements that aren't <dfn> and headings",
-   html: "<span data-dfn-type=dfn id=foo>Foo</span>",
-   changesToBaseDfn: []
-  },
-  {title: "ignores headings without a data-dfn-type",
-   html: "<h2 id=foo>Foo</h2>",
-   changesToBaseDfn: []
-  },
```

Comment:
why were these removed?

Reply:
"This update also removes duplicate dfns extraction tests"
I only removed them once, they're still in there!